style/popup.css

```diff
@@ -0,0 +1,270 @@
+.card-popup-window {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(169, 173, 175, 0.9);
+  z-index: 999;
+  display: none;
+  overflow-y: auto;
+}
+
+.popup-main-container {
+  width: 100%;
+  height: auto;
+  justify-content: center;
+  display: flex;
+}
+
+.card-popup-window.open-pop-up {
+  display: block;
+}
+
+.popup-main-container .main-container {
+  width: 80%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 2rem;
+  position: relative;
+  z-index: 9999;
+  margin: 3rem 0;
+}
+
+.card-popup-window::-webkit-scrollbar {
+  width: 12px;
+}
+
+.card-popup-window::-webkit-scrollbar-thumb {
+  background: linear-gradient(transparent, #3e4141);
+  border-radius: 6px;
+}
+
+.card-popup-window::-webkit-scrollbar-thumb:hover {
+  background: linear-gradient(transparent, #8a9dff);
+}
+
+.card-popup-window .main-container .title {
+  margin-bottom: 0.5rem;
+}
+
+.card-popup-window .main-container .title h2 {
+  font-size: 28px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.card-popup-window .main-container .info {
+  margin-bottom: 2rem;
+  height: auto;
+  color: #7a869a;
+}
+
+.card-popup-window .main-container .info span {
+  margin-right: 1rem;
+}
+
+.card-popup-window .main-container .info span:first-child {
+  color: #000;
+  font-weight: 600;
+}
+
+.card-popup-window .main-container .card-image {
+  width: 100%;
+  text-align: center;
+}
+
+.card-popup-window .main-container .card-image img {
+  max-width: 100%;
+  width: 100%;
+}
+
+.card-popup-window .main-container .card-image img:hover {
+  transform: scale(1);
+}
+
+.left-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 20px;
+}
+
+.card-popup-window .left-container .text-container {
```

Comment:
- [ ] Kindly increase the margin between the image and the project details so as to follow the Figma design.

Your design
![Capture2](https://user-images.githubusercontent.com/85095702/162264410-8885235c-2df4-464b-bf8a-ef720ce5d7a0.PNG)

Figma template

![figma](https://user-images.githubusercontent.com/85095702/162264476-df7a37bc-f97e-44cf-8183-d102e16cd7ba.PNG)